### PR TITLE
Tsconfig param

### PR DESCRIPTION
**Pull request by apiel**
Add find-up to search for tsconfig file.
Provide as well tsconfig param to be able to provide tsconfig path: `--tsconfig ./tsconfig.ts`

Still need to update doc and might require to update tests.

### CODE REVIEW

File: ts-json-schema-generator.ts [hunk 1]
```diff
@@ -6,7 +6,8 @@ import { BaseError } from "./src/Error/BaseError";
 import { formatError } from "./src/Utils/formatError";
 
 const args = commander
-    .option("-p, --path <path>", "Typescript path")
+    .option("-p, --path <path>", "Source file path")
+    .option("-f, --tsconfig <path>", "TsConfig path")
```
Comment (domoritz) [inline ts-json-schema-generator.ts]: TsConfig is not a meaningful name. Please describe.

**Comment by domoritz** [comment]
@apiel can you update the PR so we can mere it?

**Comment by apiel** [comment]
Test are passing but it might be possible to improve them for this PR. For the docs, I don't think I am the right person for that, you might have better understanding than I do about the whole project... Anyhow, I don't have time to update test and docs, sorry.

**Comment by muzea** [comment]
Can specify tsconfig is very useful to me. What is blocking this PR?

**Comment by domoritz** [comment]
See my comments above. Nothing major just a few updates. Feel free to submit a new PR with these changes in it.

**Pull request closed without merging**